build/go_check.sh

```diff
@@ -26,6 +26,15 @@ if [[ "${f}" ]]; then
   exit 1
 fi
 
+# Check dependencies up-to-date
+go mod download
+d=$(go get -u all 2>&1)
+if [[ "${d}" ]]; then
+  echo "The following dependencies are updated, please run 'go get -u all' locally and commit the change."
```

Comment:
Nit: I prefer "The following dependencies are **out of date**..."

Reply:
Done